web-registry/src/lib/rdf/rdf.compacted-context.ts

```diff
@@ -1,31 +1,57 @@
 import * as jsonld from 'jsonld';
 import { JsonLdDocument, NodeObject } from 'jsonld';
 
+import { ANCHORED_PROJECT_CONTEXT, UNANCHORED_PROJECT_CONTEXT } from './rdf';
+
 export const COMPACTED_CONTEXT = {
```

Comment:
i feel like a comment describing a little bit about what this is used for could be helpful. i.e. maybe something like:

> this context is meant to be used on any incoming JSON-LD document.
> we use this as a way to normalize the format of any incoming JSON-LD document.
> it simplifies parsing these documents in our application.

Reply:
good idea, done: https://github.com/regen-network/regen-web/pull/1701/commits/97b20a023085d5e1046baa71f1314ba464b82867